libs/sdk-ui-tests-e2e/README.md

```diff
@@ -6,37 +6,58 @@ This project contains End-to-End tests for GoodData.UI components.
 
 Tests in this library verify various go-through scenarios for GoodData.UI components. Tests are implemented using [Cypress](https://www.cypress.io/).
 There are two categories of tests: isolated and integrated. Isolated tests run against recorded backend responses, whereas integrated tests
-run against a live backend. Isolated tests run in the pre-merge phase and itegrated tests run in post-merge phase.
+run against a live backend. Isolated tests run in the pre-merge phase and integrated tests run in post-merge phase.
 
-## DEV Guide for isolated tests
+## Prepare env variables
 
-### Running tests locally
+```
+export TEST_BACKEND=https://abc.your-domain.com
+export TEST_BACKEND_NO_PREFIX=abc.your-domain.com
+export HOST=$TEST_BACKEND
+export IMAGE_ID=ui-sdk-scenarios
+
+```
+
+Provide all necessary info in the `.env` file. You can find more information in the file `.env.template`.
+
+For Tiger
```

Comment:
should mention TIGER_DATASOURCES_NAME for tiger backend. e.g: TIGER only - Set TIGER_DATASOURCES_NAME to specify data source to provide data for workspace

Reply:
I think the user can found that variable in .evn.template as I mentioned in line 21.